java/adapter/jdbc/src/test/java/org/apache/arrow/adapter/jdbc/AbstractJdbcToArrowTest.java

```diff
@@ -52,10 +52,10 @@ protected static Table getTable(String ymlFilePath, Class clss) throws IOExcepti
 
 
   /**
-   * This method creates Connection object and DB table and also populate data into table for test
+   * This method creates Connection object and DB table and also populate data into table for test.
    *
-   * @throws SQLException
-   * @throws ClassNotFoundException
+   * @throws SQLException on error
```

Comment:
Having "on error" here seems a bit unnecessary/verbose. I am slight +1 to either allowing:

```
@throws SQLException
```
or just get rid of the throws tag.

Reply:
Yeah, "on error" is a bad description but it would take to long to do it any better right now. I think this rule is mostly useful to force people to put meaningful descriptions, so I'd like to keep it.  Maybe we can improve the "on error" descriptions in the future?

Reply:
SGTM.